src/functions/delimsizing.js

```diff
@@ -145,10 +145,16 @@ defineFunction({
         // \left case below triggers parsing of \right in
         //   `const right = parser.parseFunction();`
         // uses this return value.
+        const color = context.parser.gullet.macros.get("\\current@color");
+        if (color && typeof color !== "string") {
```

Comment:
Do we give `\current@color` a default value somewhere?

Reply:
No, the default value is `undefined`, which implicitly translates to `color: inherit` in CSS (i.e., don't change the color, use whatever CSS sets it to).  Note that `rightColor` has a type of `?string`.  I'll add a comment about the meaning of `undefined`.

Reply:
Thanks!